components/rss/actions/merge-rss-feeds/merge-rss-feeds.js

```diff
@@ -0,0 +1,94 @@
+const Parser = require('rss-parser');
```

Comment:
Just yesterday, we added an `.eslintrc` file to this repo. Would you mind doing the following to apply the new formatting rules to this commit?

1. Pull the latest `master` into your fork / rebase.
2. From the root of your repo, run `npm ci` to install ESLint and related dependencies.
3. Do you work with ESLint in other repos? I like to use the VSCode ESLint plugin to autoformat my files. You can also run `eslint --fix components/rss/actions/merge-rss-feeds/merge-rss-feeds.js` to modify the file with the right formatting.